src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,265 @@
+        }
+        outputDataInConsole("For room reservation please fill out the following form");
+
+        bookRoomMenu();
+        outputSplitLine();
+        return foundRooms; 
+    }
+
+    private Long bookRoomMenu() throws IOException {
+        Long reserveId = 0L;
+        boolean bookFlaf = true;
+        while (bookFlaf) {
+            outputDataInConsole("Do you have booking room?");
+            showUserMenu("Booking menu");
+            Integer bookRoom = (Integer) inputDataFromUser("Enter number: ", true,
+                    false, false, false);
+            Integer numberRoom = null;
+            switch (bookRoom) {
+                case (1):
+                    try {
+                        numberRoom = (Integer) inputDataFromUser("Enter the room number: ", true,
+                                false, false, false);
+                    } catch (IOException e) {
+                        outputDataInConsole("Entered incorrect number room");
+                    }
+                    /*reserveId = reservationController.bookRoom(numberRoom);*/
+                    outputDataInConsole("Guest room " + reserveId.intValue() + " reserved!");
+                    outputSplitLine();
+                    break;
+                case (2):
+                    serviceMenu();
+                    bookFlaf = false;
+                    break;
+                default:
+                    bookFlaf = false;
+            }
+        }
+        return reserveId;
+    }
+
+    private List<Reservation> getAllUserReservationsMenu() {
+        outputSplitLine();
+        List<Reservation> reservList = reservationController.getAllUserReservations();
+        reservList.forEach(item -> outputDataInConsole(item.toString()));
+        return reservList;
+    }
+
+    private void cancelReservationMenu() {
+        Long idReservation = 0L;
+
+        try {
+            idReservation = (Long) inputDataFromUser("Enter your reservation number: ", false,
+                    true, false, false);
+            boolean reservId = reservationController.cancelReservation(idReservation);
+            if (reservId) {
+                outputDataInConsole("Reserve a room removed!");
+                outputSplitLine();
+            } else {
+                outputDataInConsole("Entered incorrect number reservation!");
+                cancelReservationMenu();
+            }
+        } catch (IOException e) {
+            outputDataInConsole("Room reservations missing");
+        }
+    }
+
+    private void compareDate(Date dateFrom, Date dateTo) throws IOException {
+        boolean compareDate = true;
+        while (compareDate) {
+            if (dateFrom.compareTo(dateTo) <= 0) {
+                compareDate = false;
+            } else {
+                outputDataInConsole("Room reserved at least a day!");
+                dateTo = (Date) (inputDataFromUser("Enter arrival date in the format dd.mm.yyyy: ",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "dateTo". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)